clients/web/src/app.js

```diff
@@ -25,7 +25,7 @@ import 'girder/stylesheets/layout/global.styl';
 import 'girder/stylesheets/layout/layout.styl';
 
 import 'bootstrap/js/modal';
-import 'girder/utilities/jQuery'; // $.girderModal
+import 'girder/utilities/JQuery'; // $.girderModal
```

Comment:
If we do have to do it the side-effect way, I think we ought to restructure the files so that we have one prototype method per file, so that it would be more obvious why it's getting imported, e.g.:

    import 'girder/utilities/jQuery/girderModal';

Since we won't get the tree-shaking benefits, that will at least allow us to manually select and also have a correspondence with the name of the symbol we're planning to use.